roles/openshift_health_checker/openshift_checks/logging/fluentd_config.py

```diff
@@ -0,0 +1,144 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+from openshift_checks.ovs_version import OvsVersion
+
+
+class FluentdConfig(LoggingCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
```

Comment:
@juanvallejo @sosiouxme, now that I noticed we have two checks -- `fluentd` and `fluentd_config` -- is it justifiable to have them separate? Is the distinction clear enough to explain users?